zkstore/store_test.go

```diff
@@ -0,0 +1,503 @@
+package zkstore
+
+import (
+	"crypto/md5"
+	"crypto/sha1"
+	"crypto/sha512"
+	"fmt"
+	"io"
+	"testing"
+
+	"github.com/dcos/dcos-go/testutils"
+	"github.com/samuel/go-zookeeper/zk"
+	"github.com/stretchr/testify/require"
+)
+
+// TODO: perhaps use a _buckets folder for the buckets. that would allow
```

Comment:
I forgot to remove this TODO. I removed it with an additional commit since I didn't want to rebase since I've already requested reviews.